cpp/order/HexaticTranslational.h

```diff
@@ -48,40 +53,41 @@ template<typename T> class HexaticTranslational
     void computeGeneral(Func func, const freud::locality::NeighborList* nlist,
                         const freud::locality::NeighborQuery* points, freud::locality::QueryArgs qargs);
 
-    const T m_k;
+    const T m_k;     //!< The symmetry order for Hexatic, or normalization for Translational
+    const bool m_weighted; //!< Whether to use neighbor weights in computing the order parameter (default false)
     util::ManagedArray<std::complex<float>> m_psi_array; //!< psi array computed
 };
 
-//! Compute the translational order parameter for a set of points
+//! Compute the hexatic order parameter for a set of points
```

Comment:
I swapped the order of the `Hexatic` and `Translational` classes so that it mirrors the order in the rest of the code.